compose/install/compose-other.md

```diff
@@ -0,0 +1,65 @@
+---
+description: How to install Docker Compose - Other Scenarios
+keywords: compose, orchestration, install, installation, docker, documentation
+toc_max: 3
+
+title: Other install Scenarios
+---
+
+## Install Compose Standalone
+
+### On Linux
+> **Compose standalone**
+>
+> Note that Compose standalone uses the _dash compose_ syntax instead of current's standard syntax (_space compose_).
```

Comment:
Could an example be helpful here? 

```suggestion
> Note that Compose standalone uses the _dash compose_ syntax instead of current's standard syntax (_space compose_). For example: Use `docker-compose up` for Compose standalone, instead of `docker compose up`
```

Reply:
Collapsing addressed comment.